lib/bolt/inventory/group.rb

```diff
@@ -8,7 +8,6 @@ class Group
       def initialize(data)
         @logger = Logging.logger[self]
         @name = data['name']
-
```

Comment:
You didn't like the grouping? I found it easier to read.

Reply:
Ah, I think this was an accident. I'll add it back.

Reply:
You did it one line down :)